source/AI.cpp

```diff
@@ -1248,12 +1248,12 @@ bool AI::Stop(Ship &ship, Command &command, double maxSpeed, const Point *direct
 		if(direction)
 		{
 			// Time to turn from facing backwards to target:
-            double degreesFromBackwards = TO_DEG * acos(min(1., max(-1., direction->Unit().Dot(-velocity.Unit()))));
-            double turnFromBackwardsTime = degreesFromBackwards / ship.TurnRate();
-            forwardTime += turnFromBackwardsTime;
+			double degreesFromBackwards = TO_DEG * acos(min(1., max(-1., direction.Unit().Dot(-velocity.Unit()))));
+			double turnFromBackwardsTime = degreesFromBackwards / ship.TurnRate();
+			forwardTime += turnFromBackwardsTime;
 			
 			// Time to turn from facing forwards to target:
-            double degreesFromForward = TO_DEG * acos(min(1., max(-1., direction->Unit().Dot(angle.Unit()))));
+            double degreesFromForward = TO_DEG * acos(min(1., max(-1., direction.Unit().Dot(angle.Unit()))));
```

Comment:
if your text editor is configured with tabs as 3-4 consecutive spaces, you can use its default trim trailing whitespace & spaces -> tabs macros. There are two consecutive spaces in the license header so if your editor has tabs as 2 spaces, make sure to change that back :+1: